app/controllers/users_controller.rb

```diff
@@ -0,0 +1,50 @@
+class UsersController < ApplicationController
+  before_action :set_user, except: [:index, :new, :create]
+
+  def index
+    @users = User.without_user(current_user)
```

Comment:
Would it be better if we just hide the actions for that particular user instead or provide some other indicator that that is who you're logged in as?